addons/website_hr_recruitment/data/hr_job_demo.xml

```diff
@@ -1298,37 +1363,50 @@
-                    <h2>What We Offer</h2>
-                    <br/>
-                    <p class="lead">
-                        Each employee has a chance to see the impact of his work.
-                        You can make a real contribution to the success of the company.
-                        <br/>
-                        Several activities are often organized all over the year, such as weekly
-                        sports sessions, team building events, monthly drink, and much more
-                    </p>
-                    <div class="row pt16">
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-gift rounded-circle text-bg-primary m-3"></i>
-                            <h3>Perks</h3>
-                            <p>A full-time position <br/>Attractive salary package.</p>
-                        </div>
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-bar-chart rounded-circle text-bg-primary m-3"></i>
-                            <h3>Trainings</h3>
-                            <p>12 days / year, including <br/>6 of your choice.</p>
-                        </div>
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-futbol-o rounded-circle text-bg-primary m-3"></i>
-                            <h3>Sport Activity</h3>
-                            <p>Play any sport with colleagues, <br/>the bill is covered.</p>
-                        </div>
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-coffee rounded-circle text-bg-primary m-3"></i>
-                            <h3>Eat &amp; Drink</h3>
-                            <p>Fruit, coffee and <br/>snacks provided.</p>
+                    <div class="col-lg-12">
+                        <h3>What We Offer</h3>
+                        <p class="lead">
+                            Each employee has a chance to see the impact of his work.
+                            You can make a real contribution to the success of the company.
+                            <br/>
+                            Several activities are often organized all over the year, such as weekly
+                            sports sessions, team building events, monthly drink, and much more.
+                        </p>
+                    </div>
+                    <div class="row">
+                        <div class="col-lg-3">
+                            <div class="s_hr pt-4 pb32">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                            <i class="fa fa-gift mb-3 rounded fs-5 bg-o-color-3"></i>
+                            <h3 class="h5-fs">Perks</h3>
+                            <p class="mb-0">A full-time position <br/>Attractive salary package.</p>
```

Comment:
`mb-0`, why? 🤔

Beside conflicting with the theme's general settings (not bad per-se, but we should avoid unless really necessary), applying this class will bake the text "touch" the column edge when a bg-color is applied.

---
![image](https://github.com/user-attachments/assets/28ff1f86-e464-45ba-97a6-f21fa5d256a8)

---

Also, the overall white-space management looks better without the class 👍 

| with `mb-0` | without `mb-0` |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/0bfd5f1b-b1d0-45ed-8a52-055a69dd2380) | ![image](https://github.com/user-attachments/assets/1f1499f2-6898-4e27-9823-abe3f16b13b5) |